de/_posts/2019-12-07-rails-beginner-workshop.markdown

```diff
@@ -0,0 +1,10 @@
+---
+tags: beginner rails_beginner_workshop
+categories: de events
+slug: Anfänger-Workshop
+layout: single_event
+lang: de
+i18n_key: rails_beginner_workshop_2019_12
```

Comment:
this is how jekyll knows no matter what language, that this file is linked to the other one, regardless of whether file names match or not